src/components/admin/TechnicalQuestionCard.tsx

```diff
@@ -82,15 +47,29 @@ const TechnicalQuestionCard = ({
         <HStack justify="space-between" spacing={4}>
           <HStack spacing={4} flex={1}>
             <VStack align="flex-start" spacing={2}>
-              <Text fontWeight="bold" fontSize="lg">
-                {question.title}
-              </Text>
+              <Box>
+                <ReactMarkdown components={ChakraUIRenderer()} skipHtml>
+                  {question.title || question.prompt}
+                </ReactMarkdown>
+              </Box>
               <Badge colorScheme="purple" px={2} py={1} borderRadius="full">
                 {question.topicName}
               </Badge>
             </VStack>
           </HStack>
           <HStack spacing={2}>
+            {isAdmin && (
```

Comment:
Idk if we should pass in a variable is admin instead of verifying the user is an admin.